app/assets/stylesheets/styles.scss

```diff
@@ -172,12 +266,11 @@ html {
             flex-flow: column nowrap;
             justify-content: flex-end;
             align-items: center;
-            box-shadow: inset 0 8px 10px -6px rgba(0,0,0,0.3), inset 0 8px 90px -20px rgba(0,0,0,0.15);
             width: 100%;
-            flex-grow: 1;
+            height: 272px;
             margin: 0 auto;
             padding: 0px;
-            background: $blue;
+            background: url('https://static-mh.content.disney.io/starwars/assets/footer/vader-1c0fe900f0e4.jpg') no-repeat center, #1a1a1a;
```

Comment:
Background again needs to be darker. Please choose a shade, I can make the background with that shade.